marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/matching/include-matches.xqy

```diff
@@ -19,7 +19,7 @@ return (
     test:assert-equal(1, fn:count($def-match/@threshold[. = "Definitive Match"])),
     test:assert-equal(1, fn:count($def-match/@action[. = $constants:MERGE-ACTION])),
     test:assert-exists($def-match/matches),
-    test:assert-equal(7, fn:count($def-match/matches/match))
+    test:assert-equal(8, fn:count($def-match/matches/match), "Wrong count of matches: " || xdmp:describe($def-match, (), ()) )
```

Comment:
If this test fails, it'll be very helpful to know why 8 matches are expected, and which 8 they are. Otherwise, a developer has to figure out themselves what should match and what should not.

Reply:
Added tests for the individual matches as well that makes it more clear.